airflow/providers/google/cloud/hooks/datacatalog.py

```diff
@@ -659,7 +659,7 @@ def list_tags(
         location: str,
         entry_group: str,
         entry: str,
-        page_size: int = 100,
+        page_size: Optional[int] = 100,
```

Comment:
Is it optional really ? I think it's quite ok to set it as mandatory since it has default

Reply:
We mark this as `Optional[int]` in `CloudDataCatalogListTagsOperator`, so if we don't also mark it as `Optional[int]` here, mypy errors:

```
airflow/providers/google/cloud/operators/datacatalog.py:1287: error: Argument "page_size" to "list_tags" of "CloudDataCatalogHook" has incompatible type "Optional[int]"; expected "int"
```

This is the kind of type error that mypy wasn't catching before because the decorator on the hook method dropped type information about the method, by the way.

Reply:
OK. Making MyPy happy :)